bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/handler/SensiboSkyHandler.java

```diff
@@ -0,0 +1,444 @@
+                    if (command instanceof RefreshType) {
+                        updateState(channelUID, new StringType(unit.getAcState().getMode()));
+                    } else if (command instanceof StringType) {
+                        final StringType newValue = (StringType) command;
+                        updateAcState(unit, MODE_PROPERTY, newValue.toString());
+                        addDynamicChannelsAndProperties(unit);
+                    }
+                } else if (CHANNEL_SWING_MODE.equals(channelUID.getId())) {
+                    if (command instanceof RefreshType) {
+                        if (unit.getAcState().getSwing() != null) {
+                            updateState(channelUID, new StringType(unit.getAcState().getSwing()));
+                        } else {
+                            updateState(channelUID, UnDefType.UNDEF);
+                        }
+
+                    } else if (command instanceof StringType) {
+                        final StringType newValue = (StringType) command;
+                        updateAcState(unit, SWING_PROPERTY, newValue.toString());
+                    }
+                } else if (CHANNEL_FAN_LEVEL.equals(channelUID.getId())) {
+                    if (command instanceof RefreshType) {
+                        if (unit.getAcState().getFanLevel() != null) {
+                            updateState(channelUID, new StringType(unit.getAcState().getFanLevel()));
+                        } else {
+                            updateState(channelUID, UnDefType.UNDEF);
+                        }
+                    } else if (command instanceof StringType) {
+                        final StringType newValue = (StringType) command;
+                        updateAcState(unit, FAN_LEVEL_PROPERTY, newValue.toString());
+                    }
+                } else if (CHANNEL_TIMER.equals(channelUID.getId())) {
+                    if (command instanceof RefreshType) {
+                        if (unit.getTimer() != null && unit.getTimer().secondsRemaining > 0) {
+                            updateState(channelUID, new DecimalType(unit.getTimer().secondsRemaining));
+                        } else {
+                            updateState(channelUID, UnDefType.UNDEF);
+                        }
+                    } else if (command instanceof DecimalType) {
+                        final DecimalType newValue = (DecimalType) command;
+                        updateTimer(unit, newValue.intValue());
+                    } else {
+                        updateTimer(unit, null);
+                    }
+                }
+            } else {
+                updateStatus(ThingStatus.OFFLINE);
+            }
+        } else {
+            updateStatus(ThingStatus.OFFLINE);
+        }
+    }
+
+    @Override
+    protected void updateStatus(ThingStatus status) {
+        // Do not report status if not really changed
+        if (reportedStatus != status) {
```

Comment:
The framework should check that. If not, it's broken there.

Reply:
Seems to work now